contents/docs/advanced/proxy/managed-reverse-proxy.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Managed Reverse Proxy
+sidebar: Docs
+showTitle: true
+---
+
+PostHog's Managed Reverse Proxy feature allows you to route traffic through PostHog's managed infrastructure, simplifying the deployment and management of your PostHog instance. This guide will walk you through the setup and configuration process.
+
+## Prerequisites
+
+Before setting up the Managed Reverse Proxy, ensure you have the following:
+- A PostHog Cloud account with the [Teams add-on](https://posthog.com/pricing#addons)
+- Access to your domain's DNS settings.
+
+## Creating A Managed Reverse Proxy
+
+1. Navigate to [organization proxy](https://us.posthog.com/settings/organization-proxy) settings
+2. Click "New Managed Proxy"
+3. Enter a subdomain that you control. For example if you run posthog on `myapp.com`, you could use `ph.myapp.com`
+4. Go to your DNS provider and create a new CNAME record for this subdomain
+  - You will be shown a domain to set as the CNAME record, it will look something like `4854cf84789d8596ad01.proxy-us.posthog.com.`
```

Comment:
```suggestion
4. Go to your DNS provider and create a new CNAME record:
  - The `@` or target should be set to the subdomain you chose in the previous step
  - You will be shown a domain to use as the CNAME value when creating the record. It will look something like `4854cf84789d8596ad01.proxy-us.posthog.com.`
```